src/olympia/git/management/commands/git_extraction.py

```diff
@@ -19,9 +19,13 @@
 
 log = olympia.core.logger.getLogger('z.git.git_extraction')
 
-BATCH_SIZE = 10  # Number of versions to extract in a single task.
-LOCK_NAME = 'git-extraction'  # Name of the lock() used.
-SWITCH_NAME = 'enable-git-extraction-cron'  # Name of the waffle switch.
+# Number of versions to extract in a single task. If you change this value,
+# please adjust the soft time limit of the `extract_versions_to_git` task.
+BATCH_SIZE = 10
```

Comment:
adjust the time limit where?  (I don't see one defined - somewhere adjacent to `extract_versions_to_git` anyway)

Reply:
right, I think we're using the default. I'll add a note.